Integrations/integration-GRR.yml

```diff
@@ -34,8 +34,21 @@ script:
     var server = params.server + ':' + params.port;
     var insecure = params.insecure;
 
+    var getCSRFToken = function(url) {
+        var res = http(
+                url,
+                {
+                    Method: 'GET',
+                    Username: username,
+                    Password: password
+                },
+                insecure
+            );
+        return res.Cookies[0].Value;
```

Comment:
Just add verification that the request was succeeded 
```
if (res.StatusCode < 200 || res.StatusCode >= 300) {
    throw 'Failed to get CSRF token, request status code: ' + res.StatusCode + ' and Body: ' + res.Body + '.';
}
```

Reply:
Complete